### ISSUE TITLE
1.20.1 replay recording can't load in 1.20.2/3/4

### ISSUE THREAD

**Issue by SuperNoobYT** [ReplayMod/ReplayMod#927]
When i load a replay recorded in 1.20.1 with minecraft 1.20.4 the game is stuck on the dirt screen.
the last thing to show in the log is "Loaded entity tracker in 539ms"

I've tested with 1.20.3 and 1.20.2, and the bug seem to appears post 1.20.2

My game is running with :

replaymod 2.6.14
fabric loader 0.15.3
[Upload
[replay_recordings.zip](https://github.com/ReplayMod/ReplayMod/files/13811595/replay_recordings.zip)
ing replay_recordings.zip…]()

**Comment by phuwmc** [comment]
Yep! I had this very same issue trying to run on v1.20.2 so im using 1.20 to do replay recordings now. Its quite frustrating because I rely on it for creating content.
